Updated correct batttery on electric shears

Torres84f opened the pull request:
#### Summary

Bugfixes "Electric shears now use medium batteries instead of incorrect light batteries"

Purpose of change

The old electric shears used the incorrect light_battery_cell type, which made them useless as it requires 25 charges and light battery has max 2. This fixes an oversight and makes the item functional using medium batteries.

Describe the solution

Edited pets.json to change the tool magazine

Describe alternatives you've considered

None. This is a direct fix for incorrect item configuration.

Testing

Loaded the game and spawned the electric shears. Confirmed that they now use medium batteries and can be loaded/used as expected. No errors on load.

Additional context

None.

Torres84f committed:
Updated correct batttery on electric shears pets.json

GuardianDll reviewed (changes requested):
That is not a correct fix, electric shears do not use medium batteries, they either use tool battery or they do not have any battery at all, and just connected to a grid

Torres84f committed:
Update pets.json

2nd edit, removed battery capacity added cord and power usage based on https://www.amazon.com/UILFHX-Electric-Professional-Shearing-Livestock/dp/B0CNXSSXNC

GuardianDll committed:
Lint

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

GuardianDll commented on data/json/items/tool/pets.json:
```diff
@@ -106,15 +106,9 @@
-      }
-    ],
+    "use_action": { "type": "link_up", "cable_length": 5 },
+    "charges_per_use": 1,
```
Shearing is pretty time consuming activity, and with 850 w of power draw, so something like using it for 5 minutes would cost about 250 kj of energy
```suggestion
    "charges_per_use": 250,
```

Torres84f committed:
Update data/json/items/tool/pets.json

corrected!

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>
